src/commands/create/map.ts

```diff
@@ -0,0 +1,167 @@
+import { flags as oclifFlags } from '@oclif/command';
+import { isValidIdentifier } from '@superfaceai/ast';
+import { SuperJson } from '@superfaceai/one-sdk';
+import { parseDocumentId } from '@superfaceai/parser';
+import { grey, yellow } from 'chalk';
+import { join as joinPath } from 'path';
+
+import {
+  composeUsecaseName,
+  DEFAULT_PROFILE_VERSION_STR,
+  META_FILE,
+} from '../../common';
+import { Command } from '../../common/command.abstract';
+import { developerError, userError } from '../../common/error';
+import { mkdirQuiet } from '../../common/io';
+import { formatShellLog } from '../../common/log';
+import { OutputStream } from '../../common/output-stream';
+import { createMap } from '../../logic/create';
+
+export default class CreateMap extends Command {
+  static strict = false;
+
+  static description =
+    'Creates empty map for specified profile on a local filesystem.';
+
+  static args = [
+    {
+      name: 'profileId',
+      required: true,
+      description: 'Profile Id - scope(optional) and name of a profile',
+    },
+  ];
+
+  static flags = {
+    ...Command.flags,
+    usecase: oclifFlags.string({
+      char: 'u',
+      multiple: true,
+      description: 'Usecases that map contains',
+    }),
+    provider: oclifFlags.string({
+      char: 'p',
+      description: 'Name of a Provider',
+      required: true,
+    }),
+    variant: oclifFlags.string({
+      char: 't',
+      description: 'Variant of a map',
+      dependsOn: ['provider'],
+    }),
+    template: oclifFlags.string({
+      options: ['empty', 'pubs'],
+      default: 'empty',
+      description: 'Template to initialize the usecases and maps with',
+    }),
+    scan: oclifFlags.integer({
+      char: 's',
+      description:
+        'When number provided, scan for super.json outside cwd within range represented by this number.',
+      required: false,
+    }),
+  };
+
+  static examples = [
+    '$ superface create:map sms/service -p twillio',
+    '$ superface create:map sms/service -p twillio -u SendSMS ReceiveSMS',
+  ];
+
+  private warnCallback? = (message: string) => this.log(yellow(message));
+  private logCallback? = (message: string) => this.log(grey(message));
+
+  async run(): Promise<void> {
+    const { argv, flags } = this.parse(CreateMap);
+
+    //Check input
+    const documentName = argv[0];
+
+    if (
+      documentName === 'profile' ||
+      documentName === 'map' ||
+      documentName === 'both'
+    ) {
+      throw userError('Name of your document is reserved!', 1);
```

Comment:
this seems arbitrary. i think there are some reserved names, but i'm pretty sure 'both' is not one of them ;)